docs/cep/index.md

```diff
@@ -22,11 +22,111 @@ You can process streams to perform the following actions with your data:
+
+Stream workers provide an isolated execution environment for your processing logic that allows you to deploy and execute processing logic independent of other stream workers in the system. Therefore, it's always recommended to have processing logic related to a single use case in a single StreamApp. This will help you to group processing logic and easily manage addition and removal of various use cases.
+:::
+
+
+## Stream Worker Explanation
+
+This section explains the parts of this stream worker and what they are doing.
+
+### Metadata
+
+This information defines basic information about the stream worker. Every stream worker must have at least a name and query language version in order to be valid.
+
+- **Name** - `@App:name("ExampleApp")`
+- **Query language version (optional)** - @App:qlVersion("2")
+- **Description (optional)** - @App:description('An application for enriching transactions.')
+- **Other information (optional)** - By convention, you can enter a comment with testing information, update logs, or other useful information at the beginning of the stream worker definition between `/**` and `**/`. This is similar to a docstring in functions.
+
+### Input and Output
+
+Define the input stream and the Macrometa collection that need to be joined as follows. If the stream or collection do not exist, then Macrometa creates them when you publish the stream worker.
+
+#### Define the Source Stream
+
+This stream is where the data is coming from. For more information about defining a STREAM in a stream worker, refer to [STREAM](../query-guide/stream.md) in the [Query Guide](../query-guide/index.md). For more information about streams in general, refer to [Streams](../../streams/index.md).
+
+```sql
+CREATE STREAM TransactionStream (userId long, transactionAmount double, location string);
+```
+
+#### Define the Table (Collection)
+
+`CREATE TABLE` defines where the stream worker stores your data. In this case, it will be a [Document Store Collection](../../collections/documents/index.md) For more information about defining a TABLE in a stream worker, refer to [Table (Collection)](../query-guide/table-collection.md). For more information about collections in general, refer to [Collections](../../collections/index.md).
+
+```sql
+CREATE TABLE GLOBAL UserTable (userId long, firstName string, lastName string);
+```
+
+#### Define the Sink Stream
+
+The sink stream is where the stream worker sends your data.
+
+```sql
+CREATE SINK EnrichedTransactionStream WITH (type='stream', stream='EnrichedTransactionStream', map.type='json') (userId long, userName string, transactionAmount double, location string);
+```
+
+### Data Enrichment Query
```

Comment:
Note from @diana-macrometa:

This worked specifically for the Enrich Data, but not all queries will have the pieces that were in the Enrich Data query.

The more I look at this, the more I think we need to have this narrower, except for the metadata bit. The parts we're referring to are still being rewritten.

Metadata
Source brief definition, then link
Sink brief definition, then link
Query brief definition, then link

Might add Table later, but we haven't rewritten that content yet.